### PR TITLE
fix: Akhand path view display pane issue

### PR DESCRIPTION
**Pull request by theamanjs**
Refs: #1382

### CODE REVIEW

File: www/src/scss/viewer/shabad-deck/_shabad-deck.scss [hunk 1]
```diff
@@ -4,12 +4,16 @@
     background-size: cover;
     height: unset;
     padding-bottom: 10px;
+    position: fixed;
+    overflow-y: scroll;
```
Comment (Gauravjeetsingh) [inline www/src/scss/viewer/shabad-deck/_shabad-deck.scss:8]: @theamanjs veerji, everything looks fine here. Just a minor change, can you switch up the order of CSS properties. overflow-y should be before position.

**Review by Gauravjeetsingh — CHANGES_REQUESTED** [review]

File: www/src/scss/viewer/shabad-deck/_shabad-deck.scss (hunk 1 above)
Comment (theamanjs) [reply www/src/scss/viewer/shabad-deck/_shabad-deck.scss:8]: @Gauravjeetsingh thank you for your feedback veerji, I'm doing it right now :smile:

Comment (theamanjs) [reply www/src/scss/viewer/shabad-deck/_shabad-deck.scss]: @Gauravjeetsingh veerji, I have made the changes.

**Review by Gauravjeetsingh — APPROVED** [review]

**Comment by Gauravjeetsingh** [comment]
@theamanjs Instead of sending PRs to dev branch, can you make it to **version-9-qa-fixes** branch. We will merge to dev after all fixes are complete.

**Pull request merged**